specification/devspaces/resource-manager/Microsoft.DevSpaces/stable/2019-04-01/devspaces.json

```diff
@@ -578,6 +578,19 @@
           "type": "string",
           "readOnly": true
         },
+        "hostEndpointType": {
```

Comment:
>"hostEndpointType" [](start = 8, length = 18)

An addition of a mutable property should involve a new api-version.  See the "breaking changes" doc here: [https://armwiki.azurewebsites.net/api_contracts/Intro.html](https://armwiki.azurewebsites.net/api_contracts/Intro.html)